js/id/ui/alert.js

```diff
@@ -20,12 +22,31 @@ iD.ui.Alert = function(message,type,stack) {
     if(type === 'notice'){alertDiv.classed('blue',true);}
     if(type === 'success'){alertDiv.classed('green',true);}
 
+    function modalWindow() {
+        var modalbg = d3.select('body').append('div')
+            .attr('id','detailDiv')
+            .classed('fill-darken3 pin-top pin-left pin-bottom pin-right', true);
+        var detailModal = modalbg.append('div')
+            .classed('contain col8 pad1 hoot-menu fill-white round detailModal', true);
+        detailModal.append('h1')
+            .attr('id','processingDivLabel')
+            .style({'text-align': 'center'})
+            .text('Hoot Core Command Details');
+    }
+    window.onclick = function(event) {
```

Comment:
Should we do this as a window click or a click handler on `#detailDiv`? Might make more sense to do it on `#detailDiv` since 1. it takes up the whole screen and 2. we don't have to worry about the `window.onclick` interfering with anything else in the browser.

cc @sisskind